packages/design-system/src/components/N8nMarkdown/Markdown.vue

```diff
@@ -214,7 +222,50 @@ export default {
 	}
 }
 
+.sticky {
+  color: var(--color-text-dark);
+
+  h1, h2, h3, h4 {
+    margin-bottom: var(--spacing-2xs);
+    font-weight: var(--font-weight-bold);
+  }
+
+  h1 {
+    font-size: 36px;
```

Comment:
Need to clarify this since we have not in the design system 36px; Same comment for h2 tag ```24px```